keyboards/torn/keymaps/kylejm/config.h

```diff
@@ -0,0 +1,3 @@
+#define ENCODER_RESOLUTION 2
```

Comment:
This needs a GPL license header, as well as a `#pragma once`

Reply:
Done 👍